### PR TITLE
exporter/azureblobexporter Second PR

### LINKED ISSUE #8823: New component: Azure Blob Exporter
## The purpose and use-cases of the new component

This exporter saves logs and traces to [Azure Blob Storage](https://azure.microsoft.com/services/storage/blobs/).

## Example configuration for the component
The following settings are required:

- `connection_string` (no default): Azure Blob Storage connection key, which can be found in the Azure Blob Storage resource on the Azure Portal.

The following settings can be optionally configured:

- `logs_container_name` (default = "logs"): Name of the container where the exporter saves blobs with logs.
- `traces_container_name` (default = "traces"): Name of the container where the exporter saves blobs with traces.

Example:

```yaml
exporters:
  azureblob:
    connection_string: DefaultEndpointsProtocol=https;AccountName=accountName;AccountKey=+idLkHYcL0MUWIKYHm2j4Q==;EndpointSuffix=core.windows.net
```

## Telemetry data types supported
Logs, Traces

### PR DESCRIPTION
**Pull request by eedorenko**
Description:
azureblobexporter exports logs and traces to Azure Blob Storage

Fixes #8823
Follows #11871 

Testing:
Unit tests covering exporter code

Documentation:
Readme describing exporter functionality and configuration.

### CODE REVIEW

**Comment by eedorenko** [comment]
@pmm-sumo please review

File: exporter/azureblobexporter/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Azure Blob Exporter
```
Comment (pmm-sumo) [inline exporter/azureblobexporter/README.md:1]: We should [include status header](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10116) in each component README

File: CHANGELOG.md [hunk 2]
```diff
@@ -175,6 +175,8 @@
 
 ### 🚀 New components 🚀
 
+- `azureblobexporter`: exports logs and traces to Azure Blob Storage
```
[the remark this answers is not in the document]
Comment (eedorenko) [reply CHANGELOG.md]: Done

File: exporter/azureblobexporter/README.md (hunk 1 above)
Comment (eedorenko) [reply exporter/azureblobexporter/README.md:1]: Done

**Comment by eedorenko** [comment]
Need to fix lining

**Comment by eedorenko** [comment]
> pmm-sumo is no longer involved in the project. @mx-psi as the next sponsor on the list, please review this component.

**Pull request closed without merging**